src/c++/perf_analyzer/infer_context.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright 2023-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
> @rmccorm4: 1. The implementation of `Output()` being something that only supports string outputs named "text_output" felt not ideal, but I understand the reasoning for doing it this way in terms of supporting LLM-specific behavior right now. 
    - More generically down the line, I might expect the report to include all outputs or something.

Dumping all outputs now